libswift/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+# This source file is part of the Swift.org open source project
+#
+# Copyright (c) 2014 - 2021 Apple Inc. and the Swift project authors
+# Licensed under Apache License v2.0 with Runtime Library Exception
+#
+# See http://swift.org/LICENSE.txt for license information
+# See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+
+if (NOT SWIFT_TOOLS_ENABLE_LIBSWIFT)
+  # A dummy libswift if libswift is disabled
+  add_swift_host_library(libswift OBJECT LibSwiftStubs.cpp)
+else()
+  project(LibSwift LANGUAGES C CXX Swift)
```

Comment:
You should make sure to guard this if we have CMAKE_Swift_COMPILER. And if we don't have it, abort.